j2me/communication/src/org/javarosa/service/transport/securehttp/AuthenticatedHttpTransportMessage.java

```diff
@@ -241,6 +250,30 @@ public void send() {
             }
         }
     }
+    
+    private void attemptBrowserCertificateAcceptCallout() {
+    	try {
+	    	if(CalloutResolver != null) {
```

Comment:
What situations will this be null/not null in? If its null, we can do nothing/should we notify the user?